__tests__/app.test.js

```diff
@@ -112,4 +115,45 @@ describe('GET /api/articles', () => {
             })
         })
     })
+    test("/api/articles returns status 200 and an array of article objects sorted by date in descending order", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+            const body = response.body
+            expect(body).toBeSortedBy("created_at", { descending: true })
+        })
+    })
+});
+describe('GET COMMENTS BY ARTICLE ID - /api/articles/:article_id/comments', () => {
+    test("GET /api/articles/:article_id/comments returns status 200 and an array of comment objects for the given article ID with the following properties: comment_id, votes, created_at, author, body, article_id", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+            const comments = response.body.comments
+            console.log(comments)
+            expect(Array.isArray(comments)).toBe(true)
+            expect(comments.length).toBe(11)
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number)
+                })
+            })
+        })
+    })
+    test("GET /api/articles/:article_id/comments returns comments with the most recent comments first", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+            const comments = response.body.comments
+            expect(comments).toBeSortedBy("created_at", { descending: true })
+        })
```

Comment:
3 tests missing here:
Status 200, valid ID, but has no comments responds with an empty array of comments
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999